vault/token_store.go

```diff
@@ -1237,23 +1361,32 @@ func (ts *TokenStore) RevokeTree(ctx context.Context, id string) error {
 		return fmt.Errorf("cannot tree-revoke blank token")
 	}
 
-	// Get the salted ID
-	saltedID, err := ts.SaltID(ctx, id)
+	idHMAC, err := ts.hmac(ctx, id)
 	if err != nil {
 		return err
 	}
 
 	// Nuke the entire tree recursively
-	return ts.revokeTreeSalted(ctx, saltedID)
+	err = ts.revokeTreeObfuscated(ctx, idHMAC)
+	if err != nil {
+		return err
+	}
+
+	// This is only here for backwards compatibility
```

Comment:
I don't think we would reliably hit this upgrade case. The revokeTreeOfuscated function doesn't return an error (from what I can tell) when no token is found.

Reply:
We are not relying on revokeTreeObfuscated to return an error. The intention here is to call revoke on both types of values regardless, as there really is no way for us know for sure which version to use.